pkg/stringsearch/fieldnamefinder.go

```diff
@@ -9,20 +9,20 @@ import (
 	"github.com/brimsec/zq/zng/resolver"
 )
 
-type fieldNameFinder struct {
+type FieldNameFinder struct {
 	checkedIDs       big.Int
-	fieldNameIter    fieldNameIter
-	stringCaseFinder *stringCaseFinder
+	fieldNameIter    FieldNameIter
+	stringCaseFinder *CaseFinder
```

Comment:
This is fine for now, but I'd eventually like to rename this to `caseFinder` to keep it in sync with its type.

Reply:
I'll change it.